sqlalchemy_hana/dialect.py

```diff
@@ -436,12 +438,25 @@ def visit_drop_view(self, drop: DropView | BaseDropView, **kw: Any) -> str:
             return f"DROP VIEW {self.preparer.format_table(drop.element)}"
         return f"DROP VIEW {drop.name}"
 
+    def visit_create_column(
+        self, create: CreateColumn, first_pk: bool = False, **kw: Any
+    ) -> str:
+        if create.element.autoincrement is True and not create.element.identity:
+            create.element.identity = Identity()
+        return super().visit_create_column(create, first_pk, **kw)
+
 
 class HANAExecutionContext(default.DefaultExecutionContext):
     def fire_sequence(self, seq: Sequence, type_: Integer) -> int:
         seq = self.identifier_preparer.format_sequence(seq)
         return self._execute_scalar(f"SELECT {seq}.NEXTVAL FROM DUMMY", type_)
 
+    def get_lastrowid(self) -> int:
+        self.cursor.execute("SELECT CURRENT_IDENTITY_VALUE () FROM DUMMY")
+        res = self.cursor.fetchone()
+        assert res, "No lastrowid available"
```

Comment:
Is an Assertion Error expected or could we raise a proper exception?

Reply:
I think this should never really happen, but the assert is for us and mypy